internal/k8s/client.go

```diff
@@ -362,8 +362,8 @@ func (w *ClientWrapper) patchCoreDNSConfigMap(coreDeployment *appsv1.Deployment,
 maesh:53 {
     errors
     rewrite continue {
-        name regex ([a-zA-Z0-9-_]*)\.([a-zv0-9-_]*)\.maesh maesh-{1}-{2}.maesh.svc.%[1]s
-        answer name maesh-([a-zA-Z0-9-_]*)-([a-zA-Z0-9-_]*)\.maesh\.svc\.%[2]s {1}.{2}.maesh
+        name regex ([a-zA-Z0-9-_]*)\.([a-zv0-9-_]*)\.maesh %[3]s-{1}-{2}.%[3]s.svc.%[1]s
```

Comment:
if we modify the maesh service name here, don't we also have to ensure that the Maesh Service is created with the same maesh name format?

https://github.com/containous/maesh/pull/348/files#diff-0d5beddb6a9b5c9dbe4fd5fe543b2471R273

Reply:
How would you do this ?  Writing a test ?

Reply:
Not sure. Maybe have a struct that holds the service name, and has 2 recievers, one that generates a mesh service name from the user service name, and the other that generates this regex?

That way we could test the struct to ensure that the names it generates match the regex it spits out?

Reply:
however we might want to have that in a separate PR (and out of the scope of this issue)